os_migrate/roles/validate_resource_files/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+os_migrate_validate_files: true
+es_migrate_validate_resource_files_filter:
+  - regex: .*
```

Comment:
The resource_files_filter variable can be removed i think, it's not used anywhere.